examples/todo-list/src/__tests__/integration/todo.integration.ts

```diff
@@ -0,0 +1,58 @@
+import {expect, toJSON} from '@loopback/testlab';
+import {TodoListApplication} from '../../application';
+import {TodoListRepository, TodoRepository} from '../../repositories';
+import {
+  givenRunningApplicationWithCustomConfiguration,
+  givenTodoInstance,
+  givenTodoListInstance,
+  givenTodoRepositories,
+} from '../helpers';
+
+describe('TodoListApplication', () => {
```

Comment:
And here too.

- Proposed file name: `todo-repository.integration.ts`
- Proposed suite name: `describe('TodoRepository'`